src/actions/vstudio/_vstudio.lua

```diff
@@ -338,7 +338,7 @@
 		end
 
 		-- Then the system libraries, which come undecorated
-		local system = config.getlinks(cfg, "system", "fullpath")
+		local system = config.getlinks(cfg, "system", "name")
```

Comment:
This will fail if you try to link to a library relative to the project directory: `links { "../../bin/somelib" }`. Needs to be the full path.